src/site/content/en/lighthouse-performance/third-party-facades/index.md

```diff
@@ -0,0 +1,94 @@
+
+The data about deferrable products and available facades is [maintained in third-party-web](https://github.com/patrickhulce/third-party-web/).
+
+The audit fails if the page loads resources belonging to one of these third-party embeds.
+
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./audit.jpg" 
+       alt="Lighthouse third-party facade audit highlighting Vimeo embedded player and Drift live chat.">
+  <figcaption class="w-figcaption">
+    Lighthouse third-party facade audit.
+  </figcaption>
+</figure>
+
+## How to defer third-parties with a facade
+
+Instead of adding a third-party embed directly to your HTML, load the page with a static element that looks similar to the actual embedded third-party. The interaction pattern should look something like this:
+
+1. On load: Add facade to the page.
+
+2. On mouseover: The facade preconnects to third-party resources.
+
+3. On click: The facade replaces itself with the third-party product.
+
+## Recommended facades
+
+In general, video embeds, social button widgets, and chat widgets can all employ the facade pattern. The list below offers our recommendations of open-source facades. When choosing a facade, take into account the balance between the size and feature set.
+
+### YouTube embedded player
+
+* [paulirish/lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed)
+
+* [justinribeiro/lite-youtube](https://github.com/justinribeiro/lite-youtube)
+
+* [Daugilas/lazyYT](https://github.com/Daugilas/lazyYT)
+
+### Vimeo embedded player
+
+* [luwes/lite-vimeo-embed](https://github.com/luwes/lite-vimeo-embed)
+
+* [slightlyoff/lite-vimeo](https://github.com/slightlyoff/lite-vimeo)
+
+### Live chat (Intercom, Drift, Help Scout, Facebook Messenger)
+
+* [calibreapp/react-live-chat-loader](https://github.com/calibreapp/react-live-chat-loader) ([blog post](https://calibreapp.com/blog/fast-live-chat))
+
+### Iframes
+
+* [vb/lazyframe](https://github.com/vb/lazyframe)
+
+{% Aside 'caution' %}
+
+There are some tradeoffs when lazily loading third-parties with facades as they do not have the full range of functionality of the actual embeds. For example, the Drift Live Chat bubble has a badge indicating the number of new messages. If the live chat bubble is deferred with a facade, users will not see the badge until after first interaction. For video embeds, autoplay may not work consistently if it's loaded lazily.
+
+{% endAside %}
+
+## Writing your own facade
+
+You may choose to [build a custom facade solution](https://wildbit.com/blog/2020/09/30/getting-postmark-lighthouse-performance-score-to-100#:~:text=What%20if%20we%20could%20replace%20the%20real%20widget) which employs the interaction pattern outlined above. The facade should be significantly smaller in comparison to the deferred third-party product and only include enough code to mimic the appearance of the product.
```

Comment:
Sorry for the late addition, just published the facades addition process. It would be nice to link to it in this documentation.
```suggestion
You may choose to [build a custom facade solution](https://wildbit.com/blog/2020/09/30/getting-postmark-lighthouse-performance-score-to-100#:~:text=What%20if%20we%20could%20replace%20the%20real%20widget) which employs the interaction pattern outlined above. The facade should be significantly smaller in comparison to the deferred third-party product and only include enough code to mimic the appearance of the product.

If you would like your solution to be included in the list above, check out our [submissions process](https://github.com/patrickhulce/third-party-web/blob/master/facades.md).
```

Reply:
I was going to suggest this!